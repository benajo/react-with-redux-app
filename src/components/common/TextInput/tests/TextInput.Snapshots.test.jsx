import React from 'react';
import renderer from 'react-test-renderer';
import TextInput from '../TextInput';

it('render TextInput component', () => {
  const tree = renderer.create(
    <TextInput
      name="input"
      label="Input"
      onChange={jest.fn()}
    />,
  );
  expect(tree).toMatchSnapshot();
});
