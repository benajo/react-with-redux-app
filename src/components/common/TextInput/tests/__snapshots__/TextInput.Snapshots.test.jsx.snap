// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render TextInput component 1`] = `
<div
  className="form-group"
>
  <label
    htmlFor="input"
  >
    Input
  </label>
  <div
    className="field"
  >
    <input
      className="form-control"
      name="input"
      onChange={[MockFunction]}
      placeholder=""
      type="text"
      value=""
    />
    
  </div>
</div>
`;
